### PR TITLE
Adjusts and some new translations to pt_BR

### PR DESCRIPTION
**Pull request by ginglass**
1 - Some clarification on Convensions 
2 - LoA translation to portuguese
3 - Authorization Server adjusts (I suggests do not translate terms like "claims", "acr" etc )

@tmzani, @ediemerson-br or @alex-siqueira could you check and review? 
Mantive o trecho sobre cpf/cnpj como no original em inglês, mas incluí um parágrafo que acho qestá mais alinhado ao que @alex-siqueira esclareceu. Nota que o doc permanece como antes, ou seja, com a previsão de validação se o "value" estiver especificado. O trecho sobre o AS também não faz menção ao uso dos atributos loggedUser ou businessEntity  e acho que deveria informar que esses atributos do consentimento devem ser usados. Por fim, há indicação de que na claim CNPJ o conteúdo retornado é um Array e eu acho que deveria ser um elemento só (string normal), pois é do usuário autenticado.

@RalphBragg are you sure that CNPJ claim must return an array? Seams to me that this value is related to a specific busimess account that was (or might to be) used on that specific authentication.  Did I miss something?

### CODE REVIEW

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (RalphBragg) [inline open-banking-brasil-financial-api-1_ID1-ptbr.md:41]: This is done deliberately as it is how the XML to RFC tools work (or any other RFC) so please revert.

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (RalphBragg) [inline open-banking-brasil-financial-api-1_ID1-ptbr.md:54]: This is done deliberately as it is how the XML to RFC tools work (or any other RFC) so please revert.

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (RalphBragg) [inline open-banking-brasil-financial-api-1_ID1-ptbr.md:246]: essential 'true' vs essential: true. I would recommend against specifying how it is done - you really should leave it to the base specification for someone to read. i.e 'You say, if it is marked essential as per the specification.' you don't re-define how to mark it as essential as basic mistakes occur like this.

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (RalphBragg) [inline open-banking-brasil-financial-api-1_ID1-ptbr.md:258]: Same comment as above, essentail:true has been redefined. Twice; one of which is incorrect. Don't redefine how to do this in the specification. The normative examples MUST be referred too.

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (RalphBragg) [inline open-banking-brasil-financial-api-1_ID1-ptbr.md:337]: Refer to the OIDC definition of nonce: https://openid.net/specs/openid-connect-core-1_0.html#NonceNotes if it is required however as this specification inherits everything from OIDC core it should be superfluous. e.g The requirements for nonce aren't defined in wikipedia.

**Review by RalphBragg — CHANGES_REQUESTED** [review]
Commentary provided - @alex-siqueira  @DeiGratia33. Can you please review - I would strongly suggest that the language regarding the normative version 'engilsh' be improved. As it stands, with the published version, implementers couldn't implement the published version. this pull request to improve the Brazillian translation improves a lot of terms but also highlights that key terms like the fact that the scope strings were translated when they shouldn't have been originally will make it very difficult for implementers and vendors unless they are given very firm direction regarding which version is considered 'normative'. The simple statement at the start of the document may not be strong enough.

**Comment by RalphBragg** [comment]
@DeiGratia33 @alex-siqueira - I can't assign you as reviewers of the PR can you check with Mirow as to how we can do this.

**Comment by RalphBragg** [comment]
> 1 - Some clarification on Convensions
> 2 - LoA translation to portuguese
> 3 - Authorization Server adjusts (I suggests do not translate terms like "claims", "acr" etc )
> 
> @tmzani, @ediemerson-br or @alex-siqueira could you check and review?
> Mantive o trecho sobre cpf/cnpj como no original em inglês, mas incluí um parágrafo que acho qestá mais alinhado ao que @alex-siqueira esclareceu. Nota que o doc permanece como antes, ou seja, com a previsão de validação se o "value" estiver especificado. O trecho sobre o AS também não faz menção ao uso dos atributos loggedUser ou businessEntity e acho que deveria informar que esses atributos do consentimento devem ser usados. Por fim, há indicação de que na claim CNPJ o conteúdo retornado é um Array e eu acho que deveria ser um elemento só (string normal), pois é do usuário autenticado.
> 
> @RalphBragg are you sure that CNPJ claim must return an array? Seams to me that this value is related to a specific busimess account that was (or might to be) used on that specific authentication. Did I miss something?

@ginglass  - https://openbanking-brasil.github.io/areadesenvolvedor/#qualificacao-pessoa-natural defines CNPJ ha being an array of CNPJ 14's for a juridica person which is why this is an array. As i understand it, a  user can authenticate and share accounts for organisations that have the same CNPJ8 but different CNPJ14's hence why it is an array.

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (ginglass) [reply open-banking-brasil-financial-api-1_ID1-ptbr.md]: I thought it was a typo... sorry :-) . Reverted on the next commit.

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (ginglass) [reply open-banking-brasil-financial-api-1_ID1-ptbr.md]: I agree with you. It could cause more confusion than helps. I removed this examples/tips. Would you (or/and @alex-siqueira @DeiGratia33) check?

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (ginglass) [reply open-banking-brasil-financial-api-1_ID1-ptbr.md]: Ok, Your point (normative example) is about the term MUST? As we don´t have a reference to BCP14/RFC2119, I suggest that change.. In portuguyese we have difficult to express the diference bw must (external requirement) and shall... we use "deve" to both meanings.

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (ginglass) [reply open-banking-brasil-financial-api-1_ID1-ptbr.md]: This wasn't caused by me :-) but I´ve change the reference as you suggest.

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (RalphBragg) [reply open-banking-brasil-financial-api-1_ID1-ptbr.md]: Sorry, my point about normative examples / references is that they are the example that must be followed. Redefining examples should be avoided however what you could do us flesh out more appendix items with more non normative examples as guides. Or add more guides and sections to the aspsp user guides.

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (RalphBragg) [reply open-banking-brasil-financial-api-1_ID1-ptbr.md]: Thanks

Comment (RalphBragg) [reply open-banking-brasil-financial-api-1_ID1-ptbr.md]: Also please add yourself to the contributors section. I'll add you to the English version as well when this is done. So that the next editions of both specs get the same contribution recognition. Thanks again!

**Review by ediemerson-br — APPROVED** [review]

**Comment by RalphBragg** [comment]
The version included should be fapi 1.0 - part 2 advanced. I know the naming is confusing but that part 2 is correct for the advanced profile.

File: open-banking-brasil-financial-api-1_ID1-ptbr.md
Comment (RalphBragg) [inline open-banking-brasil-financial-api-1_ID1-ptbr.md:285]: This is where it should be Part 2.

**Review by RalphBragg — CHANGES_REQUESTED** [review]
I can't comment on the language, only that 1-> 2 and its fine for me.

**Review by RalphBragg — COMMENTED** [review]
I have no further comments but can not approve the change as I don't know what the normative or other impacts might be. The sections i can review look too good me.

**Review by DeiGratia33 — APPROVED** [review]

**Review by DeiGratia33 — APPROVED** [review]

**Comment by DeiGratia33** [comment]
Changes look fine to me.

**Pull request merged**
